nemo/collections/asr/metrics/bleu.py

```diff
@@ -91,16 +94,25 @@ def __init__(
         batch_dim_index=0,
         dist_sync_on_step=False,
     ):
+        self.log_prediction = log_prediction
+        self.batch_dim_index = batch_dim_index
+        self.decoding = decoding
+        self.decode = None
+        self.multi_token = type(tokenize) is dict
+        self._tokenizer = {}
+
         super().__init__(
-            tokenize=tokenize,
+            tokenize=tokenize["default"] if self.multi_token else tokenize,
```

Comment:
why `default` here? Based on above input it can have any string in Dict for key ?

Reply:
Need to enforce a fallback tokenizer. Don't want to specify a unique tokenizer for every language.

Reply:
Actually changed this to no longer use default. Explicitly assigns the default tokenize to attribute.